roles/tas_single_node/tasks/certificates.yml

```diff
@@ -255,11 +255,61 @@
             (ec_key_cert_match_check.stdout_lines and ec_key_cert_match_check.stdout_lines[0].strip() != ec_key_cert_match_check.stdout_lines[1].strip())
         )
 
-- name: Create CTLog root
+- name: Custom CTLog Root
   when: >
-    (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_remote_ctlog_private_key) | list | length) == 0
-    or (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_remote_ctlog_public_key) | list | length) == 0
+    tas_single_node_custom_ctlog_private_keys is defined and tas_single_node_custom_ctlog_public_keys is defined and
+    (tas_single_node_custom_ctlog_private_keys | length > 0 ) and (tas_single_node_custom_ctlog_public_keys | length > 0 )
+  block:
+    - name: Create Custom CTlog Private Key
+      when: KeyVar != "" and (certs_dir_files.files | selectattr('path', 'equalto', tas_single_node_remote_ctlog_private_key) | list | length) == 0
```

Comment:
Let's use snake_case for variable naming.

Reply:
Will do

Reply:
:heavy_check_mark: